src/main/resources/locales/en/translation/display-status.json

```diff
@@ -0,0 +1,15 @@
+{
+  "completed": "Completed",
+  "submitted": "Submitted",
+  "optional": "Optional",
+  "viewed": "Viewed",
+  "important": "Important",
+  "notViewedYet": "Not viewed yet",
+  "notStartedYet": "Not started yet",
+  "notAvailableYet": "Not available yet",
+  "inProgress": "In progress",
+  "waitingForTheTribunal": "Waiting for the tribunal",
+  "submittedAndViewed": "Submitted",
```

Comment:
Any reason why these two are not similar? If not an oversight,  setting them both to "submittedAndViewed" or just "submitted"?